goblin/abc.py

```diff
@@ -52,8 +52,8 @@ def validate_vertex_prop(self, val, card, vertex_prop, data_type):
                 vp = vertex_prop(data_type, card=card)
                 vp.value = self.validate(v)
                 vertex_props.append(vp)
-            val = manager.ListVertexPropertyManager(
-                data_type, vertex_prop, card, vertex_props)
+            val = manager.ListVertexPropertyManager(data_type, vertex_prop,
```

Comment:
This is fine, but idk if it is necessary.

Reply:
`yapf` default, we can play a bit with weights if necessary